encoding.bs

```diff
@@ -1597,8 +1586,8 @@ method steps are:
   <ol>
    <li><p>Let <var>item</var> be the result of <a>reading</a> from <var>source</var>.
 
-   <li><p>Let <var>result</var> be the result of running the <a>UTF-8 encoder</a>'s <a>handler</a>
-   on <var>unused</var> and <var>item</var>.
+   <li><p>Let <var>result</var> be the result of running <var>encoder</var>'s <a>handler</a> on
```

Comment:
```suggestion
   <li><p>Let <var>result</var> be the result of <a>running</a> <var>encoder</var>'s <a>handler</a> on
```

Reply:
This does not want to invoke "run". It's the one case where neither "run" nor "process" is used. It's using run in the normal sense of the word. (Perhaps we should rename "run"...)